### PR TITLE
fixed issue with preview view corner radius

### PR DESCRIPTION
**Pull request by gumenuk39**

### CODE REVIEW

**Comment by gizmosachin** [comment]
Hi @gumenuk39, can you file an issue first with screenshots and some context on what the problem here was?

**Comment by gumenuk39** [comment]
If I use the custom size of the Color Slider. Preview view isn't round because of hardcode the 25 for width or height.
File Color Slider
rows 185 - 188
`case .horizontal where autoresizesSubviews:
				preview.bounds.size = CGSize(width: 25, height: bounds.height + 10)
			case .vertical where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.width + 10, height: 25)`
For example in demo I changed the width of color slider 
from 
`colorSlider.widthAnchor.constraint(equalToConstant: 40)`
to
`colorSlider.widthAnchor.constraint(equalToConstant: 26)`
and the result you can see on screenshot
IMAGE 2019-09-03 13:19:33

I fixed that lines to
`case .horizontal where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.height + 10, height: bounds.height + 10)
			case .vertical where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.width + 10, height: bounds.width + 10)`
and preview View is always round

File: ColorSlider.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ColorSlider'
-  s.version = '4.4'
+  s.version = '4.5'
```
Comment (gizmosachin) [inline ColorSlider.podspec]: Can you change this to 4.4.1?

File: Demo/ColorSliderDemo.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -326,7 +326,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 9BZNV2TE99;
+				DEVELOPMENT_TEAM = 6J9MZ7E65M;
```
Comment (gizmosachin) [inline Demo/ColorSliderDemo.xcodeproj/project.pbxproj]: Please revert these changes (or remove the `DEVELOPMENT_TEAM` setting from the pbxproj altogether).

File: Demo/ColorSliderDemo/DemoView.swift [hunk 3]
```diff
@@ -108,7 +108,7 @@ class DemoView: UIView {
 		NSLayoutConstraint.activate([
 			colorSlider.centerXAnchor.constraint(equalTo: centerXAnchor),
 			colorSlider.bottomAnchor.constraint(equalTo: centerYAnchor),
-			colorSlider.widthAnchor.constraint(equalToConstant: 15),
+			colorSlider.widthAnchor.constraint(equalToConstant: 40),
```
Comment (gizmosachin) [inline Demo/ColorSliderDemo/DemoView.swift]: Please revert this change, the `DemoView`'s `ColorSlider` is intentionally 15 points wide.

**Review by gizmosachin — CHANGES_REQUESTED** [review]

File: Demo/ColorSliderDemo/DemoView.swift (hunk 3 above)
Comment (gumenuk39) [reply Demo/ColorSliderDemo/DemoView.swift]: fixed

File: Demo/ColorSliderDemo.xcodeproj/project.pbxproj (hunk 2 above)
Comment (gumenuk39) [reply Demo/ColorSliderDemo.xcodeproj/project.pbxproj]: fixed

File: ColorSlider.podspec (hunk 1 above)
Comment (gumenuk39) [reply ColorSlider.podspec]: fixed

**Comment by gumenuk39** [comment]
also I found another issue:
touch isn't handling when you tap on preview view and it's position is top/bottom or left/right depends on Color slider orientation. for fixing this issue, I changed padding in hit test
was: 
`let padding = -20`
fix:
`
let padding: CGFloat
switch orientation {
     case .vertical:
         padding = -bounds.width
     case .horizontal:
         padding = -bounds.height
}
`

**Comment by gumenuk39** [comment]
@gizmosachin what about my PR? all your issues have been fixed
